handout/http/catching_rejections.md

```diff
@@ -0,0 +1,91 @@
+## Catching Rejections
+
+To catch rejections we use the subscriber's `error` and `complete` callbacks.
+
+```ts
+authenticate(username, password) {
+  const payload = {
+    username: username,
+    password: password
+  };
+
+  this.http.post(`${BASE_URL}/auth/login`, payload)
+    .map(response => response.json())
+    .subscribe(
+      authData => this.storeToken(response.id_token),
+      this.logError,
+      () => console.log('Authentication Complete')
+    );
+}
+```
+
+### Catch and Release
+
```

Comment:
I think we should add a small explanation of why this way to handle errors might be better sometimes than the regular error callback from the subscription. You can find a good reference here: https://github.com/Reactive-Extensions/RxJS/blob/master/doc/gettingstarted/errors.md#catching-errors